ui/raidboss/data/triggers/o11s.js

```diff
@@ -139,6 +225,116 @@
       alertText: {
         en: 'Opposite (Right)',
       },
+      groupTTS: {
+        en: 'opposite right',
+      },
+    },
+    // End Starboard / Laboard Stuff
+    {
+      id: 'O11S Flamethrower',
+      regex: /00:282b:Omega readies Flamethrower./,
+      infoText: {
+        en: 'Flames (rotate)',
+      },
+      tts: {
+        en: 'flames, then rotate',
+      },
+    },
+    {
+      id: '011S Electric Slide',
+      regex: /Omega readies Electric Slide./,
+      alertText: {
+        en: 'Stack',
+      },
+    },
+    {
+      id: 'O11S Blaster',
+      regex: /:3261:/,
+      tts: function(data, matches) {
+        if (data.role == 'healer') {
+          return {
+            en: 'Teather on ' + data.ShortName(matches[1]),
+            de: ' auf ' + data.ShortName(matches[1]),
+            fr: ' sur ' + data.ShortName(matches[1]),
+          };
+        }
+      },
+      groupTTS: function(data, matches) {
+        return {
+          en: 'Teather on ' + data.shortName(matches[0]),
+          de: ' auf ' + data.ShortName(matches[1]),
+          fr: ' sur ' + data.ShortName(matches[1]),
+        };
+      },
+    },
+    {
+
+      id: '011S Ferrofluid',
+      regex: / 14:3253:Level Checker starts using Ferrofluid on Level Checker/,
+      alertText: {
+        en: 'Different (out), Same (in)',
+      },
+      groupTTS: {
+        en: 'different out, same in',
+      },
+    },
+    // [20:26:01.568] 1B:105A629D:Person Name:0000:0000:00A3:0000:0000:0000: -
+    // :00A2: Positive // useful at all?
+    // Is it possible to see who is matching who?
+    {
+      id: 'O11S Pantokrator',
+      regex: /:3702:Omega readies Pantokrator/,
+      alertText: {
+        en: 'puddles, then directionals',
+      },
+    },
+    {
+      id: 'O11S Peripheral Synthesis',
+      regex: /14:324A:Omega starts using Peripheral Synthesis/,
+      preRun: function(data) {
+        data.synthCount = (data.synthCount || 0) + 1;
+      },
+      infoText: function(data) {
+        if (data.role == 'tank') {
+          return {
+            en: 'Tank LB',
```

Comment:
I don't think tank LB is really the optimal strat here, so I don't want to bake that into the triggers.

My general philosophy on triggers is to tell the player what to do, if there's only one way to do it (e.g. go left for starboard).  However, if there's not an obvious thing to do, then to tell you what you have so you can make that decision (e.g. you get the twin dive during grand octet, but it's not going to tell you where to go).

I think cactbot is probably already chattier than most folks want, so I don't want to take it any further in that direction.  I think if you want more chatty triggers for groupTTS for aoes and match yellow blue and to essentially replace somebody doing a raid calling monologue, then maybe those are things to add to your personal set of triggers.